### ISSUE TITLE
Override CallbackResolver

### ISSUE THREAD

**Issue by praswicaksono** [PHP-DI/Silex-Bridge#9]
currently in 1.3 CallbackResolver class only process valid `callable` and formatted service name `servicename:method`. Since mostly I use class name as service ID it feel weird if I pass `callable` service name. For example

``` php
$app->get('/{name}', HelloName::class)
    ->convert('req', HelloNameConverter::class . ':__invoke');
```

i have to append with ':__invoke' each callback. I proposed to override CallbackResolver so it can pass any callable service name without append ':__invoke'

**Comment by mnapoli** [comment]
Hi, this is a very good idea.

The `Silex\CallbackResolver` class could be overridden with an implementation that could call `Invoker\CallableResolver::resolve()` to resolve the callable.

We might need to keep support of the official notation though (`service:method`) so that users are not confused…

Feel free to submit a pull request.

**Comment by mnapoli** [comment]
Documentation added in f53a57173a8b085d99f4feeae4dff474e6d2b18e and I've released [1.4.0](https://github.com/PHP-DI/Silex-Bridge/releases/tag/1.4.0). Thanks again!
